pkg/cloudprovider/providers/gce/gce_addresses_fakes.go

```diff
@@ -154,3 +189,24 @@ func convertToAlphaAddress(object gceObject) *computealpha.Address {
 	}
 	return &addr
 }
+
+func convertToBetaAddress(object gceObject) *computebeta.Address {
+	enc, err := object.MarshalJSON()
+	if err != nil {
+		panic(fmt.Sprintf("Failed to encode to json: %v", err))
+	}
+	var addr computebeta.Address
+	if err := json.Unmarshal(enc, &addr); err != nil {
+		panic(fmt.Sprintf("Failed to convert GCE apiObject %v to beta address: %v", object, err))
+	}
+	return &addr
+}
+
+func (cas *FakeCloudAddressService) Print() {
```

Comment:
Is this used anywhere?

This is good for debugging, but I would suggest returning a `string` so that the caller can dump the information wherever it deems appropriate.